tensorboardX/writer.py

```diff
@@ -247,38 +257,49 @@ def __init__(self, log_dir=None, comment='', **kwargs):
             from datetime import datetime
             current_time = datetime.now().strftime('%b%d_%H-%M-%S')
             log_dir = os.path.join('runs', current_time + '_' + socket.gethostname() + comment)
+
         self.file_writer = FileWriter(logdir=log_dir, **kwargs)
-        v = 1E-12
-        buckets = []
-        neg_buckets = []
-        while v < 1E20:
-            buckets.append(v)
-            neg_buckets.append(-v)
-            v *= 1.1
+
+        # Create default bins for histograms, see generate_testdata.py in tensorflow/tensorboard
+        buckets = [1E-12 * (1.1**i) for i in range(774)]
```

Comment:
774 is magic number here. please use the original one

Reply:
Can I try benchmarking the performance and get back to you? If it is a significant performance improvement then it might be worth it.

Reply:
Has been reverted to the old version.